types/scene/wlr_scene.c

```diff
@@ -1589,6 +1589,9 @@ static void scene_entry_render(struct render_list_entry *entry, const struct ren
 			int window_corner_radius;
 			scene_get_next_sibling_geometry(node, &window_box, &window_corner_radius);
 
+			window_box.x -= data->logical.x;
+			window_box.y -= data->logical.y;
```

Comment:
Lgtm, but should we move this to the get sibling geometry function?

Reply:
Done